### ISSUE TITLE
Loss not converging

### ISSUE THREAD

**Issue by gururao001** [SeanNaren/deepspeech.pytorch#260]
Hey @SeanNaren I have trained the model for 2000 iterations and loss went below little bit and then stays around the same (100 - 200 ish) for rest of the iterations. How many iterations later did you see loss converge? 

Thanks

**Comment by ryanleary** [comment]
What dataset are you training on? What is your model architecture? What hyperparameters are you using? We would need significantly more information to be able to help you.

**Comment by gururao001** [comment]
it was error with the learning rate fixed it. My bad

**Comment by BuaaAlban** [comment]
> it was error with the learning rate fixed it. My bad

What's your learning rate?? could you please tell me ?
